redirect.install

```diff
@@ -71,3 +71,56 @@ function redirect_update_8101() {
   // Update the stored schema.
   $key_value_store_schema->set('redirect.field_schema_data.hash', $schema);
 }
+
+/**
+ * Update settings based on existing settings and Globalredirect settings.
+ */
+function redirect_update_8102() {
+  // Load default configuration.
+  $redirect_settings = \Drupal::config('redirect.settings');
+  $globalredirect_settings = \Drupal::config('globalredirect.settings');
+  $config_factory = \Drupal::configFactory();
+  $redirect = $config_factory->getEditable('redirect.settings');
+  $nonclean_to_clean = $redirect_settings->get('global_clean');
+  $admin_path = $redirect_settings->get('global_admin_paths');
+  $frontpage_redirect = $redirect_settings->get('global_home');
+  $deslash = $redirect_settings->get('global_deslash');
+
+  // If Globalredirect module exist default settings are taken.
+  if (\Drupal::moduleHandler()->moduleExists('globalredirect')) {
```

Comment:
Instead of checking the module, let's check the configuration:

if (!$globalredirect_settings->isNew()).

And then inside, after getting the values out, you can then delete() that configuration. And, you can also do the core.extension change right here in this if.